spackbot/handlers/reviewers.py

```diff
@@ -145,9 +145,12 @@ async def add_reviewers(event, gh):
     if unmaintained_pkgs:
         # Ask for maintainers
         # https://docs.github.com/en/rest/reference/issues#create-an-issue-comment
+        blame_cmds = ["$ spack blame {0}".format(pkg)
+                      for pkg in sorted(unmaintained_pkgs)]
```

Comment:
Do we need this to list *all* of them?  I think just one would suffice for most PRs.  If someone submits a PR with 100 changes it's going to get awfully long.

Reply:
I think that was why I originally just put zlib, heh. If we do add all the packages here, perhaps we can just show the first, OR  if it's absolutely necessary to include all of them, they can be included in a details with a summary (for the title) div. 

I think if a contributor is doing this for a bunch of packages, he/she will either be lazy and ignore the request all together, or maybe copy paste one line and then update the short package names via typing. Going back to copy paste every line (imho) seems harder.

Reply:
Good point. I've updated the PR to use the first unmaintained package, which should cover the most common "update a single package" case.